plugins/generic/pln/PLNGatewayPlugin.inc.php

```
@@ -21,6 +21,8 @@
 
 define('PLN_PLUGIN_PING_ARTICLE_COUNT', 12);
 
+include_once('Archive/Tar.php');
```

For consistency, shouldn't we be suppressing errors in the `include_once`?